hackvote/app/controllers/events_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EventsController < ApplicationController
+
+  NUMBER_OF_VOTES = 3
+
+  def new
+	  @event = Event.new
+	  @cur_events = Event.where( completed: false)
+	  @past_events = Event.where( completed: true)
```

Comment:
Not a huge deal here, but more efficient to get all of the `Event`s, then filter them in Ruby.